### ISSUE TITLE
Suggestion: Mono Noteblock support

### ISSUE THREAD

**Issue by Riyuukii** [UnJust1ce/Mixtape#30]
just like mono jukebox, is it possible to add an option to make noteblock sound to be mono? ty in advance! ^^

**Comment by AliahX** [comment]
yeah i'd totally add that, tho i thought that it was already in mixtape as i made it from the music type and they're in the same music type. Ill look into it next time i work on mixtape.

**Comment by Riyuukii** [comment]
alrighty, I'm looking forward for it! (including the 1.21 port hehe) :)
